tests/library/browsercontext-storage-state.spec.ts

```diff
@@ -316,3 +318,94 @@ it('should roundtrip local storage in third-party context', async ({ page, conte
   expect(localStorage).toEqual({ name1: 'value1' });
   await context2.close();
 });
+
+it('should support IndexedDB', async ({ page, server, contextFactory }) => {
```

Comment:
TODO: add IndexedDB to other tests:
- `should round-trip through the file`;
- `should preserve local storage on import/export of storage state`.